### PR TITLE
Sharpened the scalpels! (Scalpels can now be used for butchering.)

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
## Sharpened the scalpels! (Scalpels can now be used for butchering.) 

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Whisper
- add: Scalpels and their variants can now be used to butcher.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml [hunk 1]
```diff
@@ -54,6 +54,7 @@
   parent: BaseToolSurgery
   description: A surgical tool used to make incisions into flesh.
   components:
+  - type: Sharp
```
Comment (vulppine) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml:57]: maybe add `butcherDelayModifier` here? scalpels are small tools, after all

**Review by vulppine — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml:57]: I agree, will look into this!

**Comment by mirrorcult** [comment]
frankly i think its a little weird that you would be able to butcher an animal with a scalpel (thats not really what scalpels are for?) but i suppose its fine

**Comment by QuietlyWhisper** [comment]
> frankly i think its a little weird that you would be able to butcher an animal with a scalpel (thats not really what scalpels are for?) but i suppose its fine

You can already butcher with a saw/sword/etc mind you. It's an emergency case scenario! Any blade will work in bad circumstances.

**Comment by QuietlyWhisper** [comment]
Git is giving me a hard time with updating the branch. Gonna delete it, make a new branch and open a new pull request

**Pull request closed without merging**